ethereum/blockcreation/src/main/java/tech/pegasys/pantheon/ethereum/blockcreation/BlockMiner.java

```diff
@@ -80,6 +80,11 @@ public void run() {
         Thread.currentThread().interrupt();
       } catch (final Exception ex) {
         LOG.error("Block mining threw an unhandled exception.", ex);
+        try {
+          Thread.sleep(TimeUnit.SECONDS.toMillis(5));
+        } catch (final InterruptedException e) {
+          Thread.currentThread().interrupt();
```

Comment:
Apologies if I somehow introduced this into the POC, but this seems unrelated to the mark/sweep process.